src/api/goods.ts

```diff
@@ -4,16 +4,19 @@ import { Good } from '../types/Good';
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
 export function getAll(): Promise<Good[]> {
-  return fetch(API_URL)
-    .then(response => response.json());
+  return fetch(API_URL).then((response) => response.json());
 }
 
 export const get5First = () => {
-  return getAll()
-    .then(goods => goods); // sort and get the first 5
+  return getAll().then((goods) => {
+    return goods
```

Comment:
you can avoid using return if there's not extra computations in between
```suggestion
  return getAll().then((goods) => (
    goods
```